core-bundle/src/Resources/contao/templates/forms/form_password.html5

```diff
@@ -19,19 +19,3 @@
 
   <input type="password" name="<?= $this->name ?>" id="ctrl_<?= $this->id ?>" class="text password<?php if ($this->class): ?> <?= $this->class ?><?php endif; ?>" value="" autocomplete="new-password"<?= $this->getAttributes() ?>>
 <?php $this->endblock(); ?>
-
-<?php $this->block('label2'); ?>
```

Comment:
I think this template should not extend `form_row_double` anymore (topmost line, but I can't comment there)

Reply:
Removed in a95d1df23874b52fdc95fa9703e43dac010ea5a8